### PR TITLE
Reconfigure file structure for makers to incorporate id and support email

### PR DESCRIPTION
**Pull request by matthewcarlreetz**

### CODE REVIEW

File: src/makers/index.ts [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+  bobcat,
+  finestra,
+  helium,
+  longAP,
+  nebra,
+  rak,
+  syncrobit,
+}
+
+export const HotspotMakerModels = {
```
Comment (tyler-whitman) [inline src/makers/index.ts:22]: Couple things here,

1. We need to make sure we keep the original order of the list, which would be ordered by maker ID.
2. I don't see Helium in the hotspot selection list for some reason

Comment (tyler-whitman) [reply src/makers/index.ts:22]: Maybe its because the syncrobit hotspots.ts file is named Helium?

Comment (tyler-whitman) [reply src/makers/index.ts:22]: Ah yeah looks like thats why

Comment (tyler-whitman) [reply src/makers/index.ts:22]: Never mind on the ordering, Coco confirmed alphabetical is ok. Could we make sure it stays sorted no matter the order these are exported in though?

Comment (matthewcarlreetz) [reply src/makers/index.ts:22]: Great find on the syncrobit!

Comment (matthewcarlreetz) [reply src/makers/index.ts:22]: The order is currently alphabetical.
```
export const HotspotModelKeys = Object.keys(
  HotspotMakerModels,
).sort() as HotspotType[]
```

**Review by tyler-whitman — APPROVED** [review]

**Pull request merged**